index.html

```diff
@@ -504,7 +504,7 @@ <h2>Patient Portal</h2>
     <section id="appointment" class="appointment section-bg">
       <div class="container" style="border-radius: 10px;">
         <div class="section-title section-black-text">
-          <h2>Make an Appointment</h2>
+          <h2 style="color: white;">Make an Appointment</h2>

```

Comment:
```suggestion
          <h2 style="color: white;">Make an Appointment</h2>
L
```

Can this be moved to Css? Hardcoded color tag in index.html couldn't be good